src/_C018.ino

```diff
@@ -322,14 +299,65 @@ private:
     }
   }
 
+  void triggerAutobaud() {
+    if (C018_easySerial == nullptr || myLora == nullptr) return;
+    int retries = 2;
+
+    while (retries > 0 && !autobaud_success) {
+      if (retries == 1) {
+        if (_resetPin != -1) {
+          pinMode(_resetPin, OUTPUT);
+          digitalWrite(_resetPin, LOW);
+          delay(50);
+          digitalWrite(_resetPin, HIGH);
+          delay(200);
+        }
+      }
+
+      // wakeUP_RN2483 and set data rate
+      // Delay must be longer than specified in the datasheet for firmware 1.0.3
+      // See: https://www.thethingsnetwork.org/forum/t/rn2483a-problems-no-serial-communication/7866/36?u=td-er
+
+      // First set the baud rate low enough to even trigger autobaud when 9600 baud is active
+      C018_easySerial->begin(600);

```

Comment:
@TD-er  9600??

Reply:
Nope, this is meant to pull the pin down for longer than 1 byte at the current baudrate.
As it is very unlikely a baudrate lower than 9600 bps would be used, I set it to 600 bps.
So by sending a "0" it is pulling down the line for 8x 1/600th of a second, which is longer than 1 byte period (+ start/stop bits) of whatever active baudrate.

Reply:
this found first before reading discussion at link above, for me it was a typo at this time - Shift+9 is usually `(` or you eat `9`